cli/standalone/mod.rs

```diff
@@ -628,14 +648,13 @@ pub async fn run(data: StandaloneData) -> Result<i32, AnyError> {
   let cjs_tracker = Arc::new(CjsTracker::new(
     in_npm_pkg_checker.clone(),
     pkg_json_resolver.clone(),
-    CjsTrackerOptions {
-      unstable_detect_cjs: metadata.unstable_config.detect_cjs,
+    IsCjsResolverOptions {
+      detect_cjs: !metadata.workspace_resolver.package_jsons.is_empty(),
```

Comment:
Ditto - is this correct? Doesn't consult the value of `type` field

Reply:
See previous comment.